### PR TITLE
Prepare 2.0.0

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/leveldown-hyper/issues/44, closes https://github.com/Level/leveldown-hyper/issues/67

### CODE REVIEW

**Comment by ralphtheninja** [comment]
I didn't add `remark-toc` or `remark-collapse` since the README is tiny.

**Comment by ralphtheninja** [comment]
Still need to update `LICENSE.md`. Will pick the format from `level-js`.

**Comment by ralphtheninja** [comment]
Also, will remove copyright headers from c++ code.

File: README.md [hunk 1]
```diff
@@ -28,10 +31,10 @@ See the [contribution guide](https://github.com/Level/community/blob/master/CONT
 
 ## License
 
-Copyright &copy; 2012-present `leveldown-hyper` [contributors](https://github.com/level/community#contributors).
+[MIT](./LICENSE.md) © 2012-present [Contributors](./CONTRIBUTORS.md).
 
 `leveldown-hyper` is licensed under the MIT license. All rights not explicitly granted in the MIT license are reserved. See the included `LICENSE.md` file for more details.
```
Comment (vweevers) [inline README.md]: Can also remove this paragraph

**Pull request merged**